shippable/install.sh

```diff
@@ -143,5 +143,11 @@ pip3 install -q statistics numpy
 
 # mcuboot requirements
 
-pip3 install -r https://raw.githubusercontent.com/JuulLabs-OSS/mcuboot/master/scripts/requirements.txt
+pip3 install -r https://raw.githubusercontent.com/mcu-tools/mcuboot/master/scripts/requirements.txt
+
+# imgtool, needed for MCUboot to sign images (i.e. for TF-M S/NS builds)
+pip3 install -q imgtool
+
+# lpc_checksum, needed to build images for lpcxpresso11u68 boards
+pip3 install -q lpc_checksum
```

Comment:
We can remove this, it should be covered by the zephyr requirements.txt

Reply:
> We can remove this, it should be covered by the zephyr requirements.txt

Just to clarify @galak: why don't we do the same for imgtool?

One option would be to keep it here temporarily until it is passed in mcuboot requirements.txt.
But then, imgtool is used when we integrate with TFM (which uses it's own mcuboot, so then it "becomes" a requirement for zephyr)

Reply:
For now I kept imgtool, as it is not covered by neither mcuboot nor zephyr requirements.txt.

Reply:
I agree, why not add it to `requirements-extras.txt`? I mean, we are already installing it in CI via hack:
https://github.com/zephyrproject-rtos/zephyr/blob/master/.buildkite/run.sh#L58

Reply:
>Just to clarify @galak: why don't we do the same for imgtool?

`imgtool` is not in our requirements.txt today.

Reply:
> imgtool is not in our requirements.txt today.
@carlescufi i meant, why don't we add it in our requirements, as well?

Reply:
@carlescufi my comment was w/regards to lpc_checksum